Public/Src/Cache/ContentStore/Library/Stores/FileSystemContentStoreInternalChecker.cs

```diff
@@ -16,6 +16,7 @@
 using BuildXL.Cache.ContentStore.Tracing;
 using BuildXL.Cache.ContentStore.Tracing.Internal;
 using BuildXL.Cache.ContentStore.Utils;
+using static BuildXL.Cache.ContentStore.Stores.FileSystemContentStoreInternal;

```

Comment:
Please just qualify static method names with the type name. Makes code clearer.

Reply:
This was from a previous iteration, forgot to remove.